rbac/management/relation_replicator/relation_replicator.py

```diff
@@ -48,6 +48,7 @@ class ReplicationEventType(str, Enum):
     EXTERNAL_USER_UPDATE = "external_user_update"
     MIGRATE_CUSTOM_ROLE = "migrate_custom_role"
     MIGRATE_TENANT_GROUPS = "migrate_tenant_groups"
+    REMOVE_DEFAULT_BINDINGS = "remove_default_bindings"
```

Comment:
Seems this is not used anywhere

Reply:
yes it is now unnecessary as I consolidated replication here, thanks!